en-US/S.xml

```diff
@@ -81,6 +81,32 @@
       <term>secure</term>
       <listitem>
         <para>
+          Instead of stating that a product or service is "secure", describe specifically how features or qualities provide trustworthiness and resilience, and provide verifiable data where it exists.
+		  Avoid exaggerated, misleading, or unsubstantiated claims about security.
```

Comment:
I suggest adding the guidance to use a qualifier for this entry as well: "When discussing security, use qualifiers to avoid making definitive or unsubstantiated claims about security."